drracket/scribblings/drracket/menus.scrbl

```diff
@@ -543,3 +543,47 @@ brings the corresponding window to the front.
    DrRacket to other languages.}
 
 ]
+
+
+@; ----------------------------------------
+
+@section{@onscreen{Context menu}}
+
+The following additional menu items may be available when activating the context menu in an 
```

Comment:
I don't think we use the phrase "context menu" elsewhere in the documentation. In the [`racket/gui` docs](https://docs.racket-lang.org/gui/popup-menu_.html), it is called a "popup menu".

I guess a good idea is to look around in the drracket docs and see what term is used and then either change them all or change this one?

Reply:
popup is fine. 

I might be tempted to add 'also known as 'context menu' to the racket gui docs - and add an example to the gallery https://docs.racket-lang.org/gui/Widget_Gallery.html